src/client/routes/dashboard/index.tsx

```diff
@@ -1,44 +1,42 @@
 import * as React from 'react';
-import { Wrapper } from 'components/Wrapper';
 import { Button } from 'components/Button';
-
-interface IState {
-  counter: number;
-}
+import { bindActionCreators } from 'redux';
+import { connect, Dispatch } from 'react-redux';
+import { actions } from './reducer';
+import WithLoader from 'components/WithLoader';
+import DashboardInfo from './components/DashboardInfo';
+const DashboardInfoWrapper = WithLoader(DashboardInfo);
 
 interface IProps {
-  counter: number;
+  dashboard: any;
+  actions: any;
 }
 
-class Dashboard extends React.Component<IProps, IState> {
+class Dashboard extends React.Component<IProps, any> {
   constructor(props: IProps) {
     super(props);
-    this.state = {
-      counter : props.counter || 0,
-    };
   }
 
-  incrementCounter = () => {
-    this.setState((prevState: IState) => {
-      return {
-        counter : prevState.counter + 1,
-      };
-    });
+  componentWillMount() {
+    this.props.actions.fetch();
   }
 
   render() {
-    const { counter } = this.state;
-    return (
-      <Wrapper>
-        <Button
-          onClick={this.incrementCounter}
-        >
-          +1
-        </Button>
-        <span>{counter}</span>
-      </Wrapper>
-    );
+    const { loading, data } = this.props.dashboard;
+    console.log('loading', loading);
+    return <DashboardInfoWrapper loading={loading} data={data} />;
   }
 }
 
-export default Dashboard;
+const mapStateToProps = ({ dashboard }) => ({ dashboard });
+
+// TODO - need to check whether any need to be remove or not
+const mapDispatchToProps = (dispatch: Dispatch<any>) => ({
+  actions: {
```

Comment:
use simple action creator & avoid wrapping every action in actions object for simple use case